reana_client/api/client.py

```diff
@@ -289,7 +289,9 @@ def upload_file(workflow_id, file_, file_name, access_token):
             headers={"Content-Type": "application/octet-stream"},
             verify=False,
         )
-        return http_response.json()
+        if http_response.ok:
+            return http_response.json()
+        raise Exception(http_response.json().get("message"))
```

Comment:
Tested with https://github.com/reanahub/reana-workflow-controller/issues/339 all working fine. Shall we add an entry in changelog?

Reply:
Added.